presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestQueryRewriter.java

```diff
@@ -47,6 +50,7 @@
 {
     private static final String DEFAULT_PREFIX = "local.tmp";
     private static final QueryConfiguration CONFIGURATION = new QueryConfiguration(CATALOG, SCHEMA, "test_user", Optional.empty(), ImmutableMap.of());
+    private static final List<Property> TABLE_PROPERTIES_OVERRIDE = ImmutableList.of(new Property(new Identifier("retention_days"), new LongLiteral("21")));
```

Comment:
Not sure if it is possible, but maybe we should just use `test_property` for testing?